bosh/bosh_ssh.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+SCRIPT_DIR=$(cd "$(dirname $_)" && pwd)
+BOSH_PW=${BOSH_PW:-"abc"}
+BOSH=$(which bosh)
+
+if [ -z "${DEPLOYER_CONCOURSE}" ]; then
+  echo "Must provide DEPLOYER_CONCOURSE ip or hostname as \$1 or env. var"
+else
+  bosh() {
+  if [[ $1 == "ssh" ]]; then
+       shift
+       $BOSH ssh --default_password "${BOSH_PW}" \
```

Comment:
yes. for some reason bosh doesn't ask anymore for password interactively.... Also, it saves a lot of time.